src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -247,7 +247,7 @@
     <value>Avoid unsealed attributes</value>
   </data>
   <data name="AvoidConstArraysTitle" xml:space="preserve">
-    <value>Avoid constant arrays as arguments</value>
+    <value>Extract to static readonly field</value>
   </data>
```

Comment:
I meant we need to use a separate title for the fixer, lets leave `Avoid constant arrays as arguments` as is for analyzer title, please add another resource for fixer title
```xml
  <data name="AvoidConstArraysCodeFixTitle" xml:space="preserve">
    <value>Extract to static readonly field</value>
  </data>
```

Reply:
I made that change. Thank you